src/cascadia/TerminalApp/TerminalWindow.cpp

```diff
@@ -264,6 +263,8 @@ namespace winrt::TerminalApp::implementation
             {
                 _root->SetFullscreen(true);
             }
+
+            AppLogic::Current()->NotifyRootInitialized();

```

Comment:
@zadjii-msft Is this the right way to get a handle to `AppLogic`? BTW given that `App::Logic()` returns a singleton, should we maybe remove `static AppLogic logic;` from there and move it into `AppLogic::Current()`? If there's a singleton, I feel like it's more idiomatic to create it inside the class of the singleton.

Reply:
That's what I'd do yea